package.json

```diff
@@ -5,7 +5,8 @@
   "scripts": {
     "build": "webpack --mode=production",
     "serve": "webpack serve --open --mode=development",
-    "test": "jest"
+    "test": "jest",
+    "lint": "eslint './**/*.js' './**/*.ts'"
```

Comment:
nit: should we try to keep this alphabetized? optional

Reply:
done